libkbfs/conflict_resolver.go

```diff
@@ -75,19 +75,19 @@ func NewConflictResolver(
 		},
 	}
 
-	cr.startProcessing()
+	cr.startProcessing(context.Background)
```

Comment:
I was trying to avoid making it seem like the context could be canceled to stop CR processing, but I'll just name the param `baseCtx` and hope that does it.